EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -137,6 +138,7 @@
 from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
 from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
 (ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(ctppsPixelDigis, isRun3 = False )
+(ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(totemDAQMappingESSourceXML_TotemTiming, sampicSubDetId = cms.untracked.uint32(6) )
```

Comment:
```suggestion
(ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(totemDAQMappingESSourceXML_TotemTiming, sampicSubDetId = 6)
```
(This independently from the parameter being tracked or not. Again, master first)

Reply:
addressed in [deb4a01](https://github.com/cms-sw/cmssw/pull/38365/commits/deb4a010c508e4d98efedd1db44d0b917bf7c0c2)